src/app/store.ts

```diff
@@ -0,0 +1,41 @@
+import { combineReducers, configureStore } from '@reduxjs/toolkit';
+import {
+  persistStore,
+  persistReducer,
+  FLUSH,
+  REHYDRATE,
+  PAUSE,
+  PERSIST,
+  PURGE,
+  REGISTER,
+} from 'redux-persist';
+import storage from 'redux-persist/lib/storage';
+import actualCourseReducer from '../features/actualCourse';
+import actualLessonReducer from '../features/actualLesson';
+import coursesReducer from '../features/courses';
```

Comment:
I prefer when you have all reducers in the redux directory but it is up to you.

Reply:
I think it is better to seperate reducers for better code reading)